frontend/public/locales/en/translation.json

```diff
@@ -2,6 +2,7 @@
   " No status: ": " No status: ",
   " With violations: ": " With violations: ",
   " Without violations: ": " Without violations: ",
+  "\"{{name}}\" is not an existing job": "\"{{name}}\" is not an existing job",
```

Comment:
This one does not include "Ansible" even though all the others do. Also, we should probably either have separate messages for "Ansible job templates" and "Ansible workflow job templates".